bundles/go/go_completer.moon

```diff
@@ -0,0 +1,50 @@
+import app, config, interact from howl
+import ProcessBuffer from howl.ui
+import Process from howl.io
+import Matcher from howl.util
+append = table.insert
+
+install_gocode = ->
```

Comment:
Ah I forgot about `command.exec`. I'll also add that the `exec` command can let the user modify the 'go get' command if desired. To do that, it needs to be invoked this way:

```
howl.command.run 'exec go get -u -v github.com/nsf/gocode'
```